pages/learn/manage/serv-vars.md

```diff
@@ -1,48 +1,52 @@
 ---
-title: Service variables
+title: Environment and service variables
 ---
 
-# Service variables 
+# Environment and service variables 
 
-Service variables allow you to provide runtime configuration to a running service without having to modify your source code. You can use them to keep secrets and other sensitive values out of your codebase.
+Environment and service variables allow you to provide runtime configuration to one or more running services without having to modify your source code. You can use them to keep secrets and other sensitive values out of your codebase.
 
-__Note:__ Adding or modifying a service variable will cause the service to restart on any devices where the new setting is applied.
+__Note:__ Adding or modifying an environment variable will restart all services on a device. Service variables only restart the affected service.
 
-Resin.io allows you to set service variables at both the fleet and device level.
+Environment variables are accessible to all services running on a device, whereas service variables are assigned to a specific service. Both environment and service variables can be applied at the fleet or the individual device level. 
 
-__Note:__ Service variables defined in the dashboard will not apply to devices in [local mode][local-mode]. You will need to define them in your `resin-sync.yml`.
+Values defined for individual devices always override those defined for the fleet, and values defined for services generally override those defined for a device. So for any given variable, the **device service variable** will have top priority, followed by the **device environment variable**, then the **fleet service variable**, and finally the **fleet environment variable**.
```

Comment:
> and values defined for services generally override those defined for a device.

This might need some tuning to describe the that service vars are more specific for both the app&device association group. A programmer minded description I could think of would look like:
> and values defined for the service level override those defined for the respective application or device level.

I'm sure that this more confusing, but you get my point.

Reply:
@thgreasi yeah, it's a tricky statement to get correct, as fleet service variables will override fleet environment variables, but not device environment variables. I'll try another approach, because I agree it's not clear as is.